specification/securityandcompliance/resource-manager/Microsoft.SecurityAndCompliance/stable/2021-01-11/privateLinkServicesForExactDataMatchUpload.json

```diff
@@ -0,0 +1,1500 @@
+          "description": "An Azure AD object ID (User or Apps) that is allowed access to the FHIR service.",
+          "type": "string",
+          "pattern": "^(([0-9A-Fa-f]{8}[-]?(?:[0-9A-Fa-f]{4}[-]?){3}[0-9A-Fa-f]{12}){1})+$"
+        }
+      },
+      "required": [
+        "objectId"
+      ]
+    },
+    "ServiceCosmosDbConfigurationInfo": {
+      "description": "The settings for the Cosmos DB database backing the service.",
+      "type": "object",
+      "properties": {
+        "offerThroughput": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The provisioned throughput for the backing database.",
+          "readOnly": false,
+          "minimum": 400,
+          "maximum": 10000
+        },
+        "keyVaultKeyUri": {
+          "type": "string",
+          "description": "The URI of the customer-managed key for the backing database.",
+          "readOnly": false
+        }
+      }
+    },
+    "ServiceAuthenticationConfigurationInfo": {
+      "description": "Authentication configuration information",
+      "type": "object",
+      "properties": {
+        "authority": {
+          "type": "string",
+          "description": "The authority url for the service",
+          "readOnly": false
+        },
+        "audience": {
+          "type": "string",
+          "description": "The audience url for the service",
+          "readOnly": false
+        },
+        "smartProxyEnabled": {
+          "type": "boolean",
+          "description": "If the SMART on FHIR proxy is enabled",
+          "readOnly": false
+        }
+      }
+    },
+    "ServiceCorsConfigurationInfo": {
+      "description": "The settings for the CORS configuration of the service instance.",
+      "type": "object",
+      "properties": {
+        "origins": {
+          "type": "array",
+          "description": "The origins to be allowed via CORS.",
+          "readOnly": false,
+          "items": {
+            "$ref": "#/definitions/ServiceCorsConfigurationOriginEntry"
+          }
+        },
+        "headers": {
+          "type": "array",
+          "description": "The headers to be allowed via CORS.",
+          "readOnly": false,
+          "items": {
+            "$ref": "#/definitions/ServiceCorsConfigurationHeaderEntry"
+          }
+        },
+        "methods": {
+          "type": "array",
+          "description": "The methods to be allowed via CORS.",
+          "readOnly": false,
+          "items": {
+            "$ref": "#/definitions/ServiceCorsConfigurationMethodEntry"
+          }
+        },
+        "maxAge": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The max age to be allowed via CORS.",
+          "readOnly": false,
+          "minimum": 0,
+          "maximum": 99999
+        },
+        "allowCredentials": {
+          "type": "boolean",
+          "description": "If credentials are allowed via CORS.",
+          "readOnly": false
```

Comment:
>"readOnly": false [](start = 10, length = 17)

This is the default value, make sure to annotate your properties as readOnly : true for those set by the service that are not meant to be set by an end-user